docs/data/material/integrations/styled-components/styled-components.md

```diff
@@ -102,8 +102,8 @@ We provide boilerplate examples of Create React App with Material UI and styled
 
 <!-- #target-branch-reference -->
 
-- [Material UI + CRA + styled-components (JavaScript)](https://github.com/mui/material-ui/tree/master/examples/material-ui-cra-styled-components)
-- [Material UI + CRA + styled-components (TypeScript)](https://github.com/mui/material-ui/tree/master/examples/material-ui-cra-styled-components-ts)
+- [Material UI + Vite + styled-components (JavaScript)](https://github.com/mui/material-ui/tree/master/examples/material-ui-vite-styled-components)
+- [Material UI + Vite + styled-components (TypeScript)](https://github.com/mui/material-ui/tree/master/examples/material-ui-vite-styled-components-ts)
```

Comment:
These examples are removed. We should remove this whole `Ready-to-use examples` section from the docs.

Reply:
Yes, I removed them from the docs page as well.

![image](https://github.com/user-attachments/assets/fc0bcca9-2d22-486d-81a0-f0a87460af19)